mbed_tools_ci_scripts/create_news_file.py

```diff
@@ -0,0 +1,67 @@
+"""Easy news files generation.
+
+Usage:
+    create-news-file "Fixed a bug" --type bugfix
+"""
+import argparse
+import enum
+import pathlib
+from datetime import datetime
+
+from mbed_tools_ci_scripts.utils.configuration import configuration, ConfigurationVariable
+
+
+NEWS_DIR = configuration.get_value(ConfigurationVariable.NEWS_DIR)
+
+
+class NewsType(enum.Enum):
+    """Describes the type of news we're writing."""
+
+    bugfix = 0
+    doc = 1
+    feature = 2
+    major = 3
+    misc = 4
+    removal = 5
+
+
+def create_news_file(news_text: str, news_type: NewsType) -> pathlib.Path:
+    """Facilitates creating a news file, determining it's file name based on the type."""
+    file_path = determine_news_file_path(news_type)
+    _write_file(file_path, news_text)
+    return file_path
+
+
+def determine_news_file_path(news_type: NewsType) -> pathlib.Path:
+    """Returns an available file path for given news type."""
+    news_file_name = _determine_todays_news_file_name()
+    news_file_path = pathlib.Path(NEWS_DIR, f"{news_file_name}.{news_type.name}")
+    inc = 0
+    while news_file_path.exists():
+        inc += 1
+        news_file_path = news_file_path.with_name(f"{news_file_name}{inc:0=2}.{news_type.name}")
```

Comment:
So the initial newsfile will be `newsfilename00.bugfix` for example?

Reply:
Oh I see, by the time it reaches here it will already by `01`

Reply:
No, it'll be `foo.bugfix` then `foo01.bugfix` then `foo02.bugfix`.

Reply:
Yep, I see now 👍